packages/jest-dom-mocks/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [2.9.1] - 2020-05-05
```

Comment:
I think Quilt packages avoid version bumps for doc only changes.  Because it causes a lot of unnecessary version bumps in consumers.

Reply:
Yes, you can probably remove the entry in the changelog and simply update the doc.